.github/workflows/gh-pages-deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy on GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: npm install, npm run build
+      run: |
+        npm install
+        npm build:production
```

Comment:
Ah we forgot to use an `npm run build:production`

Reply:
Ah true